tools/indexer/example/src/main.rs

```diff
@@ -176,15 +176,16 @@ async fn listen_blocks(mut stream: mpsc::Receiver<near_indexer::BlockResponse>)
         //         ),
         //     ],
         // }
-        eprintln!("{:#?}", block);
+        // eprintln!("{:#?}", block);
+        eprintln!("Example: block height {:?}", block.block.header.height);
```

Comment:
1. It seems that there is too much data in the response, so you prefer not to print everything. I suggest we still print useful information here, so anyone who runs the example can immediately benefit from the example: block height and hash, the number of chunks, the number of transactions, the number of receipts, and the number of execution outcomes.
2. `block.block` looks suboptimal. Please, have a background thinking process on how to disabiguate these.